### ISSUE TITLE
[iceprod2] user proxies (Trac #1452)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1444]
User proxy (depends on #841):

* Store a proxy under that user's account in the DB.
  * A moderately long-lived proxy with ability to generate sub-proxies. Probably need some sort of script to make this for the user, from a certificate.
  * Make limited sub-proxy to send with tasks by this user.
  * Notify (email?) a user if proxy will expire soon and tasks are still running. 

Migrated from https://code.icecube.wisc.edu/ticket/1452
```json
{
    "status": "closed", 
    "changetime": "2016-05-09T21:55:16", 
    "description": "User proxy (depends on #847):\n\n* Store a proxy under that user's account in the DB.\n  * A moderately long-lived proxy with ability to generate sub-proxies. Probably need some sort of script to make this for the user, from a certificate.\n  * Make limited sub-proxy to send with tasks by this user.\n  * Notify (email?) a user if proxy will expire soon and tasks are still running. ", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "wontfix", 
    "_ts": "1462830916934929", 
    "component": "iceprod", 
    "summary": "[iceprod2] user proxies", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-11-25T20:09:34", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160509T21:38:10: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "wontfix"
* david.schultz commented:

> moved to https://github.com/WIPACrepo/iceprod/issues

**Comment by icecube-trac** [comment]
Trac update at 20160509T21:55:16: 
* david.schultz changed milestone from "IceProd 2.1" to ""
* david.schultz commented:

> Milestone IceProd 2.1 deleted
